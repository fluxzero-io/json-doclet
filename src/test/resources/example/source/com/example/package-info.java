/**
 * Sample package used in smoke tests.
 */
package com.example;
