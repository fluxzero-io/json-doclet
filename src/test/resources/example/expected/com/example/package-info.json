{
  "name" : "example",
  "qualifiedName" : "com.example",
  "documentation" : " Sample package used in smoke tests.\n"
}
